ui/components/ui/list-item/list-item.stories.js

```diff
@@ -31,74 +48,81 @@ Currencies.propTypes = {
 const okColor = '#2F80ED';
 const failColor = '#D73A49';
 
-export const SendComponent = () => (
+export const SendComponent = (args) => (
   <ListItem
     icon={<Send color={okColor} size={28} />}
     titleIcon={<Preloader size={16} color="#D73A49" />}
-    title={text('title', 'Send DAI')}
+    {...args}
```

Comment:
The spread of the args is the pattern that has been followed for the storybooks more generally, which is why the approach has been used here.  We should try and be consistent one way or the other with this :).  I don't mind changing, but would probably defer to leaving it with the spread for consistency.

Reply:
I hear you on being consistent, and I think it's a good rule of thumb. The thing is, most cases in our storybook that use `{...args}` use it in a way that maps the properties directly onto the opponent as expected.  I would prefer reverting the `{...args}` in this file in favor of how it was written before because it's been said that passing unnecessary props using the spread operator to react components is an anti-pattern. Another reason is that we are planning to convert more files to TypeScript. If we added TypeScript here, this should give us an error warning.